doc/api/url.md

```diff
@@ -234,6 +234,16 @@ url.resolve('/one/two/three', 'four')         // '/one/two/four'
 url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
+Note that `url.resolve()` method behaves differently depending on the protocol given in the `from` parameter. Some protocols like https, file, ftp, wss and gopher have special cases.
+
+For example:
+
+```js
+url.resolve('https://foo.tld', 'bar');   // 'https://foo.tld/bar'
+url.resolve('wss://foo.tld', 'bar')      // 'wss://bar'
+url.resolve('ftps://foo.tld', 'bar')     // 'ftps://bar'
+```
+Although,the uniform behaviour of the method is achieved when the URL ends with a slash.
```

Comment:
There’s a space missing after `Although,` (and maybe a blank line above this one, too). Also, it might be good to have an example of that behaviour listed here?